docs/getting-started/use-cases/computervision/annotation/cvat/cvat_customizing_workspace_template.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Customizing CVAT Workspace Template
+sidebar_label: Environment variables - CVAT Workspace
+description: List of CVAT Workspace environment variables.
```

Comment:
Change this to: "Onepanel - Customizing CVAT Workspace Template"

Reply:
Agreed